nvflare/ha/overseer/utils.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+from datetime import datetime, timedelta
+
+OVERSEER_STORE = os.environ.get("OVERSEER_STORE")
+
+if OVERSEER_STORE == "REDIS":
+    from .redis_store import do_refresh, get_all_sp, get_primary_sp, get_sp_by, update_sp
+elif OVERSEER_STORE == "SQL":
+    from .sql_store import do_refresh, get_all_sp, get_primary_sp, get_sp_by, update_sp
+elif OVERSEER_STORE == "MEM":
+    from .mem_store import do_refresh, get_all_sp, get_primary_sp, get_sp_by, update_sp
+else:
+    print("Using default STORE (MEM)")
+    from .mem_store import do_refresh, get_all_sp, get_primary_sp, get_sp_by, update_sp  # noqa
```

Comment:
What about making these XXX_store inherits from a common class?
What is the advantage / disadvantage of doing this VS that?